src/Workleap.Extensions.OpenAPI/RequiredType/ExtractRequiredAttributeFromNullableType.cs

```diff
@@ -25,4 +28,32 @@ public void Apply(OpenApiSchema schema, SchemaFilterContext context)
             schema.Required.Add(property.Key);
         }
     }
+
+    private static void ApplyNullabilityOnNestedSchemaProperties(OpenApiSchema schema, SchemaFilterContext context)
+    {
+        var nullabilityInfoContext = new NullabilityInfoContext();
+        var contextProperties = context.Type.GetProperties();
+        foreach (var (name, property) in schema.Properties)
```

Comment:
I am not familiar with the feature of NullabilityInfoContext is there a link we could add as comment in code that explain this concept?